zendesk/ticket_form.go

```diff
@@ -45,6 +45,7 @@ func (z Client) CreateTicketForm(ticketForm TicketForm) (TicketForm, error) {
 	var data, result struct {
 		TicketForm TicketForm `json:"ticket_form"`
 	}
+	data.TicketForm = ticketForm
```

Comment:
OMG...
I'm so sorry. It's my fault :bowing_man: 

I enbugged at https://github.com/nukosuke/go-zendesk/commit/bf9b44c6ae0ce2785f48ba328192fc5de90cba56. But, CI failed to detect this bug because mock API returns static response. We need to fix all functions for other resources...

Anyway, good catch. Thank you!